rusk-recovery/src/state.rs

```diff
@@ -45,210 +45,198 @@ pub static FAUCET_KEY: Lazy<PublicSpendKey> = Lazy::new(|| {
 });
 
 fn generate_transfer_state(
+    session: &mut Session,
     snapshot: &Snapshot,
-    state: &mut NetworkState,
-) -> Result<TransferContract, Box<dyn Error>> {
-    let mut transfer: TransferContract =
-        state.get_contract_cast_state(&rusk_abi::transfer_contract())?;
+) -> Result<(), Box<dyn Error>> {
     let theme = Theme::default();
 
     snapshot.transfers().enumerate().for_each(|(idx, balance)| {
-        info!("{} balance #{}", theme.action("Generating"), idx);
+        info!(
+            "{} balance #{} = {:?}",
+            theme.action("Generating"),
+            idx,
+            balance.notes
+        );
 
         let mut rng = match balance.seed {
             Some(seed) => StdRng::seed_from_u64(seed),
             None => StdRng::from_entropy(),
         };
 
+        info!("pushing notes {}", balance.notes.len());
```

Comment:
Same as above, IIRC this was a `println!`. Removing?

Reply:
Removed